README.md

```diff
@@ -140,7 +148,17 @@ This repository has the following structure:
   └── tailwind.config.js          # TailwindCSS configuration file
 ```
 
-## Contributors
+## Connect with us 
+
+<p align="left">
+<a href="https://asyncapi.slack.com/" target="blank"><img align="center" src="https://img.icons8.com/color/48/null/slack-new.png" alt="asyncapi" height="30" width="40" /></a>
+<a href="https://twitter.com/asyncapispec" target="blank"><img align="center" src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/twitter.svg" alt="asyncapispec" height="30" width="40" /></a>
+<a href="https://www.linkedin.com/company/asyncapi" target="blank"><img align="center" src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/linked-in-alt.svg" alt="asyncapi" height="30" width="40" /></a>
```

Comment:
```suggestion
<a href="https://www.linkedin.com/company/asyncapi" target="blank"><img align="center" src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/linked-in-alt.svg" alt="LinkedIn AsyncAPI" height="30" width="40" /></a>
```

Reply:
it should be `AsyncAPI LinkedIn` , right? @alequetzalli